### ISSUE TITLE
vndr deletes vendor files needed by golang code

### ISSUE THREAD

**Issue by andrewhsu** [LK4D4/vndr#10]
From the [docker repo](https://github.com/docker/docker), running `vndr` ends up with many files deleted from `vendor` dir which are needed at compile time:

```
$ git clone https://github.com/docker/docker.git && cd docker
$ go get github.com/LK4D4/vndr
$ vndr # usually fails on first try with ssl error
...
2016/12/27 16:21:50 	Finished clone google.golang.org/api
2016/12/27 16:21:50 Errors on clone:
golang.org/x/sys: Err: exit status 128, out: Cloning into '/Users/andrewhsu/work/src/github.com/docker/docker/vendor/golang.org/x/sys'...
fatal: unable to access 'https://go.googlesource.com/sys/': Server aborted the SSL handshake

golang.org/x/crypto: Err: exit status 128, out: Cloning into '/Users/andrewhsu/work/src/github.com/docker/docker/vendor/golang.org/x/crypto'...
fatal: unable to access 'https://go.googlesource.com/crypto/': Server aborted the SSL handshake

google.golang.org/cloud: Err: exit status 128, out: Cloning into '/Users/andrewhsu/work/src/github.com/docker/docker/vendor/google.golang.org/cloud'...
fatal: unable to access 'https://code.googlesource.com/gocloud/': Server aborted the SSL handshake
$ vndr # second try usually succeeds
$ git status | grep deleted | wc -l # count deleted files
     221
```

**Comment by LK4D4** [comment]
@andrewhsu do you by any chance have symlinks in your developer workflow? Like symlink to GOPATH or symlink to docker project?

**Comment by andrewhsu** [comment]
No symlinks used to point to the docker/docker working directory, but there are symlinks within the docker/docker working directory:
```
$ find . -type l
./bundles/1.14.0-dev/dynbinary-client/docker
./bundles/1.14.0-dev/dynbinary-daemon/dockerd
./bundles/latest
./hack/make/.build-deb/docker-engine.docker.default
./hack/make/.build-deb/docker-engine.docker.init
./hack/make/.build-deb/docker-engine.docker.upstart
./hack/make/.build-deb/docker-engine.udev
./project/CONTRIBUTORS.md
```

**Comment by LK4D4** [comment]
@andrewhsu this is weird indeed. I'm not sure what can also affect that. Maybe Go version. We can try to debug issue together, because I can't reproduce :/

**Comment by andrewhsu** [comment]
I did not have the docker repo cloned into a subdir of `GOPATH` before I ran `vndr`; it was in a totally different directory. Once I had docker cloned into `$GOPATH/src/github.com/docker/docker` executing `vndr` worked just fine (without deleting stuff from `vendor` dir).

Should it be assumed that the directory that `vndr` is executed is always within the subdir of `GOPATH`? If so, should it be checked by `vndr` upon execution or in documentation?

I'm chalking up the `Server aborted the SSL handshake` error to my ISP or some other networking configuration that may be weird at my house in New Zealand. When I run `vndr` from an EC2 instance in California I never get those SSL errors.

**Comment by LK4D4** [comment]
Yes, it should be in GOPATH. I will add that into README. 
I wonder if ssl problems can be because of high  concurrency. Vndr just runs git and hg commands in parallel. I will try to add concurrency limit, not sure if it'll help.

**Comment by LK4D4** [comment]
@andrewhsu I've merged #12 which sets parallelism limit. I wonder if it helps with SSL handshake stuff.
Thanks!

**Comment by LK4D4** [comment]
I think this was fixed.
